### ISSUE TITLE
assignment not working on transposedDataFrame

### ISSUE THREAD

**Issue by drighelli** [Bioconductor/S4Vectors#93]
Hi,

I loaded a DataFrame from a xls file and then I transposed it.

While trying to assign single/multiple values to one/more cells I get "this" error:

`Error in `[<-`(`*tmp*`, 1, 3, value = 1) : invalid subsetting`

Here it is a reproducible example:

```
score <- c(1L, 3L, NA)
df <- DataFrame(score) # single column
df <- t(df)
df[1,3] <- 1
```

Is it a desired behaviour?

Thanks,
Dario

**Comment by drighelli** [comment]
Thanks Marcel,

nice trick. 

If it's not useful to leave this issue open for further implementations we can close it.

Dario

**Comment by hpages** [comment]
Sounds like a good idea to implement a `[<-` method for these objects. Will try to get to it soon. Let's keep this issue open in the meantime. Thanks!

H.

**Comment by hpages** [comment]
Subassignment of TransposedDataFrame objects is now implemented in **S4Vectors** 0.33.9 (see commit b3bc6a4630f9f1a7713e0002618f8489f2edaf4e):
```
library(S4Vectors)

score <- c(1, 3, NA)

tdf <- t(DataFrame(score))
tdf
# TransposedDataFrame with 1 row and 3 columns
#                  1  2  3
# score <numeric>  1  3 NA

tdf[1, 3] <- 0
tdf
# TransposedDataFrame with 1 row and 3 columns
#                 1 2 3
# score <numeric> 1 3 0
```
H.
